src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,11 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService serviceInform = new UserService();
+        User user = new User(email, password);
```

Comment:
we should construct user from given fields, it's useless, just use those argument directly.
save result of call to the service to this user local variable instead